test/dependency-manager/v1-spec/environment.test.ts

```diff
@@ -0,0 +1,113 @@
+import { expect } from 'chai';
```

Comment:
Can we create an explicit validation.test.ts file?
- Test for a variety of expected validation failures
- Test for typos at various places in the env/service config